src/ClientData/include/ClientData/TimerData.h

```diff
@@ -29,13 +29,14 @@ class TimerData final : public TimerDataInterface {
   [[nodiscard]] std::vector<const orbit_client_protos::TimerInfo*> GetTimers(
       uint64_t min_tick = std::numeric_limits<uint64_t>::min(),
       uint64_t max_tick = std::numeric_limits<uint64_t>::max()) const override;
+  // This optimized method avoids returning two timers that map to the same pixel in the screen, so
+  // is especially useful when there are many timers in the screen (zooming-out for example).
+  // It assures to return at least one timer per occupied pixel. The overall complexity is
+  // O(log(num_timers) * resolution) where resolution is the number of pixels_width.
```

Comment:
Nit: This is copying all the comments from `TimerDataInterface`.
For the interface, it would be better that the comments do not mention the implementation details. 
See go/cstyle#function_declarations for the suggestion for function comments.

Reply:
offline suggestion: "efficient instead of useful"

Reply:
Thanks for the link, I refactored several comments.